src/main/java/org/jabref/logic/exporter/ExporterFactory.java

```diff
@@ -0,0 +1,117 @@
+        exporters.add(new TemplateExporter(Localization.lang("HTML list"), "listrefs", "listrefs", "listrefs", FileType.HTML, layoutPreferences, savePreferences));
+        exporters.add(new TemplateExporter(Localization.lang("HTML table (with Abstract & BibTeX)"), "tablerefsabsbib", "tablerefsabsbib", "tablerefsabsbib", FileType.HTML, layoutPreferences, savePreferences));
+        exporters.add(new TemplateExporter("Harvard RTF", "harvard", "harvard", "harvard", FileType.RDF, layoutPreferences, savePreferences));
+        exporters.add(new TemplateExporter("ISO 690 RTF", "iso690rtf", "iso690RTF", "iso690rtf", FileType.RTF, layoutPreferences, savePreferences));
+        exporters.add(new TemplateExporter("ISO 690", "iso690txt", "iso690", "iso690txt", FileType.TXT, layoutPreferences, savePreferences));
+        exporters.add(new TemplateExporter("Endnote", "endnote", "EndNote", "endnote", FileType.TXT, layoutPreferences, savePreferences));
+        exporters.add(new TemplateExporter("OpenOffice/LibreOffice CSV", "oocsv", "openoffice-csv", "openoffice", FileType.CSV, layoutPreferences, savePreferences));
+        exporters.add(new TemplateExporter("RIS", "ris", "ris", "ris", FileType.RIS, layoutPreferences, savePreferences).withEncoding(StandardCharsets.UTF_8));
+        exporters.add(new TemplateExporter("MIS Quarterly", "misq", "misq", "misq", FileType.RTF, layoutPreferences, savePreferences));
+        exporters.add(new BibTeXMLExporter());
+        exporters.add(new OpenOfficeDocumentCreator());
+        exporters.add(new OpenDocumentSpreadsheetCreator());
+        exporters.add(new MSBibExporter());
+        exporters.add(new ModsExporter());
+
+        // Now add custom export formats
+        exporters.addAll(customFormats.values());
+
+        return new ExporterFactory(exporters);
+    }
+
+    public static ExporterFactory create(JabRefPreferences preferences, JournalAbbreviationLoader abbreviationLoader) {
+        Map<String, TemplateExporter> customFormats = preferences.customExports.getCustomExportFormats(preferences, abbreviationLoader);
+        LayoutFormatterPreferences layoutPreferences = preferences.getLayoutFormatterPreferences(abbreviationLoader);
+        SavePreferences savePreferences = SavePreferences.loadForExportFromPreferences(preferences);
+        return create(customFormats, layoutPreferences, savePreferences);
+    }
+
+    /**
+     * Build a string listing of all available exporters.
+     *
+     * @param maxLineLength The max line length before a line break must be added.
+     * @param linePrefix    If a line break is added, this prefix will be inserted at the beginning of the next line.
+     * @return The string describing available exporters.
+     */
+    public String getExportersAsString(int maxLineLength, int firstLineSubtraction, String linePrefix) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
Since you are already touching this code, please also use proper naming, e.g. `builder`.